psi4/src/psi4/libfock/CompositeJK.cc

```diff
@@ -39,6 +39,7 @@
 #include "psi4/liboptions/liboptions.h"
```

Comment:
The diff interpreted DFJCOSK as being moved to CompositeJK.cc. A lot of this is renaming DFJCOSK functions to CompositeJK functions, but I will point things out where they appear.